pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -98,7 +98,7 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 	var clb *awstasks.ClassicLoadBalancer
 	var nlb *awstasks.NetworkLoadBalancer
 	{
-		loadBalancerName := b.GetELBName32("api")
+		loadBalancerName := b.CLBName32("api")
```

Comment:
we use `loadBalancerName` in both the CLB and NLB tasks, so maybe the `CLBName32` function name could be updated to reflect that?

Reply:
Done. Renamed to `LBName32`.